### ISSUE TITLE
forceNiceScale not working since V2.1.0 

### ISSUE THREAD

**Issue by DevoliaEsp** [RomRider/apexcharts-card#738]
**Checklist**
- [x] I updated the card to the latest version available
- [x] I cleared the cache of my browser
- [x] I verified that I'm really running the lastest version in my browser console
- [x] I checked if there is another issue opened with the same problem

**Describe the bug**
Since apex V2.1.0 and later, it seems that "forceNiceScale", an yaxis motion listed in apex_config, simply doesn't work anymore

**Version of the card**
Version: 2.1.2

**To Reproduce**
This is the configuration I used:
`type: custom:apexcharts-card
update_interval: 15min
apex_config:
  xaxis:
    axisBorder:
      show: false
  yaxis:
    - decimalsInFloat: 0
      forceNiceScale: true
    - decimalsInFloat: 0
      opposite: true
      forceNiceScale: true
  grid:
    show: false
  legend:
    show: false
  fill:
    type: gradient
    gradient:
      type: vertical
      opacityFrom: 0.7
      opacityTo: 0.3
header:
  show: true
  title: Salon
  show_states: true
  colorize_states: true
series:
  - entity: sensor.purificateur_d_air_temperature
    type: area
    color: '#ff444f'
    fill_raw: last
    name: Température
    group_by:
      duration: 30min
      func: avg
  - entity: sensor.purificateur_d_air_humidity
    type: area
    color: '#00b4fb'
    fill_raw: last
    name: Humidité
    group_by:
      duration: 30min
      func: avg`

**Screenshots**
![image](https://github.com/user-attachments/assets/02cf5780-6a10-4540-ab62-d17e68d54610)
![image](https://github.com/user-attachments/assets/33ec9ec9-c4f0-4a69-b1aa-cc40f2a36a09)
Before (V2.0.4) and after (2.1.0/2.1.2)

**Desktop (please complete the following information):**
 - Browser : firefox
 - Version : 128.0

**Comment by mikeynp** [comment]
+1 same for me. Reverted back to 2.0.4 and everything works ok again. For me, `forceNiceScale: true` seems to not work when I have multiple series (assigned to different y-axes), e.g.
```
series:
  - entity: sensor.thing
    name: Thing
    yaxis_id: first
  - entity: sensor.thing2
    name: Thing2
    yaxis_id: second
yaxis:
  - id: first
    apex_config:
      forceNiceScale: true
  - id: second
    apex_config:
      forceNiceScale: true
```

**Comment by ausfas** [comment]
Same here , it's fine in 2.04

**Comment by marekpsiuk** [comment]
Same for he. I had to revert to 2.0.4 to have forceNiceScale work as expected. I also have multiple series (assigned to different y-axes).

My config (works on 2.0.4 and stops working since 2.1.0).

```
type: custom:apexcharts-card
graph_span: 2h
header:
  show: true
  title: Power RAW last 1h
  show_states: true
  colorize_states: true
apex_config:
  chart:
    height: 250px
  legend:
    show: false
  yaxis:
    forceNiceScale: true
yaxis:
  - id: power
    apex_config:
      forceNiceScale: true
  - id: percent
    opposite: true
    decimals: 0
    min: 0
    max: 100
    apex_config:
      tickAmount: 5    
series:
  - entity: sensor.house_consumption
    curve: smooth
    type: line
    stroke_width: 1
    yaxis_id: power
    extend_to: false
  - entity: sensor.pv_power
    curve: smooth
    type: line
    stroke_width: 1
    yaxis_id: power
    extend_to: false
  - entity: sensor.meter_active_power_total
    curve: smooth
    type: line
    stroke_width: 1
    yaxis_id: power
    extend_to: false
  - entity: sensor.battery_power
    curve: smooth
    type: line
    stroke_width: 1
    yaxis_id: power
    extend_to: false
  - entity: sensor.energy_monitoring_smartplug_moc
    curve: stepline
    type: line
    stroke_width: 1
    yaxis_id: power
    extend_to: false
  - entity: sensor.battery_state_of_charge
    curve: stepline
    type: line
    stroke_width: 1
    yaxis_id: percent
    extend_to: false

```

I've looked into the source code changes and the only thing that could have impact was update to `"apexcharts": "^3.49.2"` 🤔

**Comment by ausfas** [comment]
Still no update on this big 😔

**Comment by Triple-S** [comment]
I'm afraid this is not a bug but a feature in the underlying apexcharts library, so it probably cannot be fixed in this repository. Here it is stated that this behaviour is probably intended although I don't understand the reason for it: https://github.com/apexcharts/apexcharts.js/issues/4283#issuecomment-1971274210
If you still want this to be fixed you probably need to open an issue here: https://github.com/apexcharts/apexcharts.js/issues

**Comment by ausfas** [comment]
not sure why its as intended where 2.04 the logic works but now it doesn't... sorry I disagree.

**Comment by Triple-S** [comment]
@ausfas It is not a bug because the logic behind forceNiceScale was changed intentionally in the apexcharts library. Whether thats good or bad is another question, but as I stated this repository is the wrong place to discuss this, because this repository will never be able to solve the problem. That is because this repository serves as a bridge between Home Assistant and Apexcharts, so anything related directly to Apexcharts, like this problem, has nothing to do with this repository.
